terraform/prod/terragrunt.hcl

```diff
@@ -20,8 +20,16 @@ generate "prod_tfvars" {
   aws_security_group_app = "App_sg"
   subnet_app_a = "App_Prod_aza_net"
   subnet_app_b = "App_Prod_azb_net"
+  frontend_logout_chain_url = "https://logon7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/logout?redirect_uri="
   front_end_redirect_path = "https://fam.nrs.gov.bc.ca"
   local_frontend_redirect_path = "http://localhost:5173"
+  fam_callback_urls = [
+    "https://fam.nrs.gov.bc.ca/authCallback",
+    "https://oidcdebuggersecure-3d5c3f-dev.apps.silver.devops.gov.bc.ca/"
+  ]
+  fam_logout_urls = [
+    "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://dev.loginproxy.gov.bc.ca/auth/realms/standard/protocol/openid-connect/logout?redirect_uri=https://fam.nrs.gov.bc.ca",
+  ]
```

Comment:
Hi Catherine, this "https://dev.loginproxy....." looks like is for "**dev**" and should not be in /**prod**/terragrunt.hcl.
Also, not sure if "https:/logon**test**7.gov.bc.ca" is wrong in /prod/terragrunt.hcl.
Could you verify and modify it to be production environment?
Thank you Catherine.